evaporate.js

```diff
@@ -1437,8 +1437,8 @@
                     InvocationType: 'RequestResponse',
                     Payload: JSON.stringify({
                         to_sign: makeStringToSign(authRequester),
-                        sign_params: makeSignParamsObject(me.signParams),
-                        sign_headers: makeSignParamsObject(me.signHeaders)
+                        sign_params: makeSignParamsObject(con.signParams),
```

Comment:
See above, we need to sue `me.signParams`. I addressed this in a subsequent commit.